rpcs3/rpcs3qt/skylander_dialog.cpp

```diff
@@ -162,6 +221,60 @@ const std::map<const u16, const std::string> list_skylanders = {
     {3013, "Grim Creeper"},
     {3014, "Rip Tide"},
     {3015, "Punk Shock"},
+	{3220, "Jet Stream"},
+	{3221, "Tomb Buggy"},
+	{3222, "Reef Ripper"},
+	{3223, "Burn Cycle"},
+	{3224, "Hot Streak"},
+	{3225, "Shark Tank"},
+	{3226, "Thump Truck"},
+	{3227, "Crypt Crusher"},
+	{3228, "Stealth Stinger"},
+	{3231, "Dive Bomber"},
+	{3232, "Sky Slicer"},
+	//{3233, "Clown Cruiser"},
```

Comment:
Why are some entries commented out

Reply:
They only work on Nintendo systems... so I left them there for completion sake, but they wouldn't actually work in RPCS3.